documentation/staging/content/managing-domains/accessing-the-domain/status-conditions.md

```diff
@@ -200,3 +201,28 @@ The following is a list of condition types for a cluster resource.
         server pods is fewer than the number specified in `cluster.spec.maxUnavailable`.
     * The `status` attribute is set to `False` if fewer than the sufficient number of WebLogic Server
       pods are ready in the cluster, or servers are rolling/starting, or a failure has occurred.
+
+### Notes on conditions
+
```

Comment:
Minor: IMO, it'd help to break this up into 3 easily located sections instead of generically calling it Notes on Conditions. Something like:

\### Condition life cycle
....
\### Conditions and generations
....
\### Conditions and events